pxtcompiler/emitter/languageservice.ts

```diff
@@ -76,6 +76,14 @@ namespace ts.pxtc.service {
         return [...retApis, ...completionSymbols(enumVals, COMPLETION_DEFAULT_WEIGHT)]
     }
 
+    export function tsKeywordToPyKeyword(keyword: string): string | undefined {
+        const map: { [tsType: string]: string } = {
```

Comment:
```suggestion
        const map: pxt.Map<string> = {
```

nit, we have our map type we use all over~